universal/iservices.md

```diff
@@ -1,10 +1,119 @@
 # Fixing iMessage and other services with OpenCore
 
-This page is for those having iMessage and other iServices issues, this is a very basic guide so will not go as in-depth into the issues as some other guides. This specific guide is a translation and reinterpretation of the AppleLife Guide on fixing iServices: [Как завести сервисы Apple - iMessage, FaceTime, iCloud](https://applelife.ru/posts/727913)
+This page is for those having iMessage and other iServices issues, this is a very basic guide so will not go as in-depth into the issues as some other guides. This specific guide is a translation and reinterpretation of the AppleLife Guide on fixing iServices: [Как завести сервисы Apple - iMessage, FaceTime, iCloud](https://applelife.ru/posts/727913).
+
+**Your Apple ID is the single most influential factor in using iServices.**
+
+If you have existing Apple products in your account, such as an iPhone, you should have no issues whatsoever using a generated serial set. However, if you recently created an account, that does not have any existing Apple hardware or App Store purchases, you may be required to call Apple once you have attemped logging in.
+
+The following items will be created below and are required to use iServices:
+
+- MLB
+- ROM*
+- SystemProductName
+- SystemSerialNumber
+- SystemUUID
+
+\**for ROM, we use the MAC Address of the network interface, lowercase, and without `:`.*
 
 **Note**: You and you alone are responsible for your AppleID, read the guide carefully and take full responsibility if you screw up. Dortania and other guides are not held accountable for what **you** do.
 
-## Generate a new Serial
+## Generate a new Serial and Board Serial (MLB) for your model.
+
+Compile [macserial](https://github.com/acidanthera/OpenCorePkg/tree/master/Utilities/macserial) from source.
```

Comment:
Instead of saying compile for all users, mention Linux users must compile. Otherwise, simply use release to avoid issues

Reply:
I only now noticed the release download, was it always there?

Reply:
Ah the problem with latest is I don't know how to refer to latest.tar.gz I think that's why I compiled it

Reply:
@khronokernel maybe we should make utilities compile on Linux as well? Could you and @dhinakg look at it? Will require a Linux C toolchain running in macOS, but that should be easier than I think. I have some scripts on how to make it if necessary.

Reply:
@sickcodes I would till like this updated for users to use release by default and mention edge case for Linux users that require compile